drakrun/drakrun/main.py

```diff
@@ -210,22 +213,22 @@ def _get_dll_run_command(pe_data):
 
         for export in exports:
             if export[1] == 'DllRegisterServer':
-                return 'start regsvr32 %f'
+                return 'regsvr32 %f'
 
             if 'DllMain' in export[1]:
-                return 'start rundll32 %f,{}'.format(export[1])
+                return 'rundll32 %f,{}'.format(export[1])
 
         if exports:
             if exports[0][1]:
-                return 'start rundll32 %f,{}'.format(export[1].split('@')[0])
+                return 'rundll32 %f,{}'.format(export[1].split('@')[0])
             elif exports[0][0]:
-                return 'start rundll32 %f,#{}'.format(export[0])
+                return 'rundll32 %f,#{}'.format(export[0])
 
         return None
 
     @staticmethod
     def _get_office_file_run_command(extension, file_path):
-        start_command = ['start']
+        start_command = ['cmd.exe', '/C', 'start']
```

Comment:
looks like `start` is making exhaustive search over `C:\Program Files` (and maybe some other locations too), we should avoid using it whenever possible

Reply:
checked, this is reading from registry HKCR